index.html

```diff
@@ -2,19 +2,449 @@
 </head>
 <body>
-    <header></header>
-    <section></section>
-    <section></section>
-    <section></section>
-    <section></section>
-    <section></section>
-    <section></section>
-    <footer></footer>
+    <header>
+        <div class="container">
+            <div class = "head flex-row">
+                <img src="img/Logo.png" alt="logoThere" class = "logo">
+                <div class="empty"></div>
+                <nav>
+                    <ul class = "menu flex-row">
+                        <li><a href="#">Home</a></li>
+                        <li><a href="#">Domain</a></li>
+                        <li><a href="#">Hosting</a></li>
+                        <li><a href="#">Clients</a></li>
+                        <li><a href="#">Blog</a></li>
+                        <li><a href="#">Support</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <div class = "title flex-column">
+
+                <div class = "flex">
+                    <h1>
+                        HELLO!
+                    </h1>
+                </div>
+                <div class="title_text flex-column">
+                    <div>We love our work...</div>
+                    <div>And We're taking it seriously...</div>
+                </div>
+            </div>
+        </div>
+        <div class="registerrow flex-row">
+            <div class="space-left"></div>
+            <div class="login flex">
+                <a href="#">Login</a>
+            </div>
+            <div class = "register flex">
+                <a href="#">Register</a>
+            </div>
+            <div class="space-right"></div>
+        </div>
+    </header>
+    <section class="Domain">
+        <div class="container2 flex-column">
+            <div class="domainText flex-column">
+                <h2 class = "flex">
+                    Would you like to buy a new domain ?
+                </h2>
+                <p class = "flex">
+                    Lorem ipsum dolor sit amet, consectetur dipiscing elit. Suspendisse auctor mauris ac nulla interdum, id molestier.
+                </p>
+            </div>
+            <div class="domainName flex-row">
+                <div class="thirdlevel flex">
+                    <input type="text" id="tld" name="DomainThirdLevel" placeholder="www.">
+                </div>
+                <div class="secondlevel flex">
+                    <input type="text" id="sld" name="DomainSecondLevel" placeholder="What do you want?">
+                </div>
+                <div class="firstlevel flex">
+                    <select class="firstlevelselect" name="DomainFirstLevel" id="fld">
+                        <option value="com">.com</option>
+                        <option value="net">.net</option>
+                    </select>
+                </div>
+            </div>
+            <div class="domaintable">
+                <table class = "flex-column">
```

Comment:
You shouldn't mix `table` and `flex layout` in one. Use either table with it's defaulted `display: table` or divs with flex.